modules/ROOT/nav.adoc

```diff
@@ -59,11 +59,11 @@
 
 .UI
 * xref:amp-react:index.adoc[React]
-** xref:amp-react:1-overview.adoc[Overview]
```

Comment:
@mchacong shouldn't we let the Overview page active in the index ?

Reply:
no longer need this